### PR TITLE
docs: Autocomplete widget does not update its displayed value

### PR DESCRIPTION
**Pull request by jeromecambon**
Covers [UID-733](https://bonitasoft.atlassian.net/browse/UID-733)

[UID-733]: https://bonitasoft.atlassian.net/browse/UID-733?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by benjaminParisel** [comment]
image

I am not satisfied with the result. I suggest to move the troubleshooting section in the dedicated widget part.

I push a commit to do it, feel free to rollback if you think it's better before.

**Comment by jeromecambon** [comment]
@benjaminParisel 
I think that including the troubleshooting in the widget description make the page less readable, and would "pollute" the widget descriptions. However, adding a pointer from the widget description to the dedicated troubleshooting (if any) would provide a good alternative, and avoid to miss it. What do you think?

**Comment by benjaminParisel** [comment]
> @benjaminParisel I think that including the troubleshooting in the widget description make the page less readable, and would "pollute" the widget descriptions. However, adding a pointer from the widget description to the dedicated troubleshooting (if any) would provide a good alternative, and avoid to miss it. What do you think?

Do what you think is better and more readable but please, don't put three troubleshooting titles and icon (like we can see on the screen)

**Comment by jeromecambon** [comment]
> > @benjaminParisel I think that including the troubleshooting in the widget description make the page less readable, and would "pollute" the widget descriptions. However, adding a pointer from the widget description to the dedicated troubleshooting (if any) would provide a good alternative, and avoid to miss it. What do you think?
> 
> Do what you think is better and more readable but please, don't put three troubleshooting titles and icon (like we can see on the screen)

I kept the same formatting already in place, but yes we should use the example we provided here: https://bonitasoft.github.io/bonita-documentation-theme/troubleshooting.html

File: modules/pages-and-forms/pages/widgets.adoc [hunk 1]
```diff
@@ -42,6 +42,8 @@ Limitations:
 * The URL should have the file type as extension (e.g. \https://myhost/doc/sample.pdf)
 ====
 
+See [#Troubleshooting UI Designer widgets] to solve common issues with this widget.
```
Comment (benjaminParisel) [inline modules/pages-and-forms/pages/widgets.adoc]:
```suggestion
See <<troubleshooting-ui-designer-widgets>> to solve common issues with this widget.
```

Example could be found here, https://bonitasoft.github.io/bonita-documentation-theme/links.html#_anchors_to_navigate_inside_the_same_document

**Review by benjaminParisel — CHANGES_REQUESTED** [review]

**Review by benjaminParisel — APPROVED** [review]

**Pull request merged**